code/model/DNProject.php

```diff
@@ -286,6 +287,20 @@ public function canDownloadArchive($member = null) {
 	}
 
 	/**
+	 * This is a permission check for the front-end only.
```

Comment:
By "front-end" I mean outside of the admin section of the site. The admin section still uses the normal canCreate().